h2o-admissibleml/src/main/java/hex/Infogram/EstimateCMI.java

```diff
@@ -0,0 +1,44 @@
+package hex.Infogram;
+
+import water.MRTask;
+import water.fvec.Chunk;
+import water.fvec.Frame;
+
+public class EstimateCMI extends MRTask<EstimateCMI> {
+  public int _nonZeroRows;
+  public double _accumulatedCMI;
+  public double _meanCMI;
+  public final double _scale = 1.0/Math.log(2);
```

Comment:
this could be static (and also applied only in postGlobal) as opposed to on every row